test/fixtures/mson-resource-nested-mixin.json

```diff
@@ -0,0 +1,266 @@
+{
+  "_version": "3.0",
+  "metadata": [],
+  "name": "",
+  "description": "",
+  "element": "category",
+  "resourceGroups": [
+    {
+      "name": "",
+      "description": "",
+      "resources": [
+        {
+          "element": "resource",
+          "name": "",
+          "description": "",
+          "uriTemplate": "/",
+          "model": {},
+          "parameters": [],
+          "actions": [
+            {
+              "name": "",
+              "description": "",
+              "method": "GET",
+              "parameters": [],
+              "attributes": {
+                "relation": "",
+                "uriTemplate": ""
+              },
+              "content": [],
+              "examples": [
+                {
+                  "name": "",
+                  "description": "",
+                  "requests": [],
+                  "responses": [
+                    {
+                      "name": "200",
+                      "description": "",
+                      "headers": [
+                        {
+                          "name": "Content-Type",
+                          "value": "application/json"
+                        }
+                      ],
+                      "body": "",
+                      "schema": "",
+                      "content": [
+                        {
+                          "element": "object",
+                          "content": [
+                            {
+                              "element": "member",
+                              "content": {
+                                "key": {
+                                  "element": "string",
+                                  "content": "b"
+                                },
+                                "value": {
+                                  "element": "object",
+                                  "content": [
+                                    {
+                                      "element": "member",
+                                      "content": {
+                                        "key": {
+                                          "element": "string",
+                                          "content": "c"
+                                        },
+                                        "value": {
+                                          "element": "extend",
+                                          "content": [
+                                            {
```

Comment:
This is the object you have here:

```
{
  "element": "object",
  "content": [
    {
      "element": "member",
      "content": {
        "key": {
          "element": "string",
          "content": "b"
        },
        "value": {
          "element": "object",
          "content": [
            {
              "element": "member",
              "content": {
                "key": {
                  "element": "string",
                  "content": "c"
                },
                "value": {
                  "element": "extend",
                  "content": [
                    {
                      "element": "object",
                      "content": [
                        {
                          "element": "object",
                          "content": [
                            {
                              "element": "object",
                              "meta": {
                                "ref": "X"
                              },
                              "content": [
                                {
                                  "element": "member",
                                  "content": {
                                    "key": {
                                      "element": "string",
                                      "content": "id"
                                    },
                                    "value": {
                                      "element": "string",
                                      "content": "pavan"
                                    }
                                  }
                                }
                              ]
                            }
                          ]
                        }
                      ]
                    }
                  ]
                }
              }
            }
          ]
        }
      }
    }
  ]
}
```

I think the mixin is nested too deeply. This is the object I am thinking of:

```
{
  "element": "object",
  "content": [
    {
      "element": "member",
      "content": {
        "key": {
          "element": "string",
          "content": "b"
        },
        "value": {
          "element": "object",
          "content": [
            {
              "element": "member",
              "content": {
                "key": {
                  "element": "string",
                  "content": "c"
                },
                "value": {
                  "element": "extend",
                  "content": [
                    {
                      "element": "object",
                      "content": [
                        {
                          "element": "object",
                          "meta": {
                            "ref": "X"
                          },
                          "content": [
                            {
                              "element": "member",
                              "content": {
                                "key": {
                                  "element": "string",
                                  "content": "id"
                                },
                                "value": {
                                  "element": "string",
                                  "content": "pavan"
                                }
                              }
                            }
                          ]
                        }
                      ]
                    }
                  ]
                }
              }
            }
          ]
        }
      }
    }
  ]
}
```